sys/targets/targets.go

```diff
@@ -715,12 +714,21 @@ func initTarget(target *Target, OS, arch string) {
 	for i := range target.CFlags {
 		target.replaceSourceDir(&target.CFlags[i], sourceDir)
 	}
-	if OS == Linux && arch == runtime.GOARCH {
-		// Don't use cross-compiler for native compilation, there are cases when this does not work:
-		// https://github.com/google/syzkaller/pull/619
-		// https://github.com/google/syzkaller/issues/387
-		// https://github.com/google/syzkaller/commit/06db3cec94c54e1cf720cdd5db72761514569d56
-		target.Triple = ""
+	if OS == Linux {
+		if cc := os.Getenv("SYZ_CC_" + OS + "_" + arch); cc != "" {
+			target.CCompiler = cc
```

Comment:
It looks like `target.CCompiler = os.Getenv("SYZ_CC_" + OS + "_" + arch)` do the same but takes 1 line instead of 3.

Reply:
It is not the same, because `os.Getenv` returns an empty string if the respective environment variable is not set, and in this case we do not want to override `target.CCcompiler`.